src/Integration.Vsix/InfoBar/InfoBarManager.cs

```diff
@@ -176,18 +210,6 @@ private static InfoBarModel CreateModel(string message, IEnumerable<string> butt
                 isCloseButtonVisible: true);
         }
 
-        private static bool TryAddInfoBarToFrame(IVsWindowFrame frame, IVsUIElement uiElement)
-        {
-            IVsInfoBarHost infoBarHost;
-            if (TryGetInfoBarHost(frame, out infoBarHost))
-            {
-                infoBarHost.AddInfoBar(uiElement);
-                return true;
-            }
-
-            return false;
-        }
-
```

Comment:
No longer required. The `TryGetInfoBarHost` is still used, but the common low-level implementation method is now responsible for adding the info bar to the host.